API-strategie-modulen/API-strategie-mod-geo/request-response.md

```diff
@@ -58,17 +48,25 @@ A simple spatial filter can be supplied as a bounding box. This is a common way
   </p>
   <h4 class="rulelab">How to test</h4>
   <ul>
-    <li>Issue an HTTP GET request to the API, including the <code>bbox</code> query parameter.</li>
-    <li>Validate that a document was returned with a status code 200.</li>
+    <li>Issue an HTTP GET request to the API, including the <code>bbox</code> query parameter and <code>bbox-crs</code> header containing a CRS supported by the provider. If the provider supports CRS84 then the <code>bbox-crs</code> header may be omitted.</li>
+    <li>Validate that a response with status code 200 is returned.</li>
     <li>Verify that only features that have a spatial geometry that intersects the bounding box are returned as part of the result set.</li>
   </ul>
 </div>
 
+TO DO 
+Bovenstaande voorbeeld is redundant met het voorbeeld bij API-GEO-10. Het lijkt me goed om hier het voorbeeld van bbox te gebruiken en bij API-GEO-10 een ander voorbeeld, of vanuit API-GEO-10 te verwijzen naar dit voorbeeld of andersom.
+
 <aside class="note">
 Spatial filtering is an extensive topic. There are use cases for geospatial operators like <code>intersects</code> or <code>within</code>. Geospatial filters can be large and complex, which sometimes causes problems since <code>GET</code> may not have a payload (although supported by some clients). 
 
 More complex spatial filtering is not addressed in this module. A new API Design Rules module on filtering will address spatial as well as non-spatial filtering. [[ogcapi-features-3]] will provide input for this.
 
+TO DO 
+Moet hier een requirement worden opgenomen, dat als er met geometrie gezocht wordt, tenminste GeoJSON <code>Geometry</code> objecten ondersteund moeten worden? Of als de geometrie niet in GeoJSON past (curves etc.) dat dan WKT en WKB ondersteund moeten worden?
```

Comment:
het zou kunnen - is ergens wel logisch om toe te voegen - ik aarzel wel om nog dingen toe te voegen.

Reply:
Ik heb een opmerking in #536 opgenomen.